integrationtests/src/test/java/io/kroxylicious/proxy/filter/ForwardContext.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter;
+
+import org.apache.kafka.common.protocol.ApiMessage;
+
+public record ForwardContext(KrpcFilterContext filterContext, FilterContext constructionContext, ApiMessage body) {
```

Comment:
`ForwardContext` doesn't really mean anything to me. Maybe `ForwardingContext` or `PropagationContext` or even `DelayedConstructionContext`

Reply:
renamed to `ForwardingContext` to match `ForwardingStyle`